src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -224,6 +224,7 @@ class PopoverReportActionContextMenu extends React.Component {
                 reportID={this.state.reportID}
                 reportAction={this.state.reportAction}
                 isArchivedRoom={this.props.isArchivedRoom}
+                isChronosReport={this.props.isChronosReport}
```

Comment:
Will you please add `isChronosReport` and `isArchivedRoom` in `propTypes` and `defaultProps`?